utils/node_local_test.py

```diff
@@ -2735,18 +2763,18 @@ def test_uns_basic(self):
         dfuse.start('uns-1')
 
         # List the root container.
-        print(os.listdir(join(dfuse.dir, pool, container)))
+        print(os.listdir(join(dfuse.dir, pool, container.uuid)))
```

Comment:
`uuid` or `id()`?

Reply:
uuid.  This is used for dfuse path lookup which is uuid based.

Reply:
No.  We've discussed it but probably not going to add it either as there are very few use-cases where dfuse isn't passed a container on the command line.